openff/toolkit/tests/test_molecule.py

```diff
@@ -159,69 +160,80 @@ def is_three_membered_ring_torsion(torsion):
 # FIXTURES
 # =============================================================================================
 
+if OpenEyeToolkitWrapper.is_available():
+    openeye_toolkit = OpenEyeToolkitWrapper()
+else:
+    openeye_toolkit = None
+
+# During import stage, read the Tripos records in memory but don't
+# process them. When requested (via .get_molecule()) use OEChem to
+# convert the record to a molecule. Cache the molecule for later
+# use. Only return copies, as the test code may modify the molecule.
+
+# NOTE: this differs somewhat from the SDF reader in test_toolkits.py:
+#   - This reads a .mol2 file, not an .sdf
+#   - get_molecule() always uses OpenEyeToolkitWrapper to parse
+#       the record, and always return a copy of the parsed record;
+
+
+class DrugBankRecord:
+    def __init__(self, name, record):
+        self.name = name
+        self.record = record
+        self._molecule = None  # Cache
+
+    def get_molecule(self):
+        if self._molecule is None:
+            with NamedTemporaryFile(mode="w+b", suffix=".mol2") as tmpfile:
+                tmpfile.write(self.record)
+                tmpfile.flush()
+                molecules = openeye_toolkit.from_file(
+                    tmpfile.name,
+                    "sdf",
```

Comment:
That should be "mol2" not "sdf". It's a copy&paste error from similar code in test_toolkits.py. The error was hidden because prior to PR# 1006 the file_format parameter wasn't converted to the appropriate OEFormat_* and passed to `ifs.SetFormat()`. This told OEChem to use the default file type detection, based on the filename extension. The fix in that PR is at https://github.com/openforcefield/openff-toolkit/pull/1006/files#diff-e2e1869da0d73e84cb7e83cc8c464ef6bfe93f5d1b075de6ed3a21e212853a70R294 .